integration-tests/utils/integration-utils.js

```diff
@@ -107,6 +107,17 @@ const findInDataTableRow = (page, entityName, text) => {
   return findElement(page, `//*[@role="grid"]//*[contains(.,"${entityName}")]/following-sibling::*[contains(.,"${text}")]`)
 }
 
+const openError = async page => {
+  //close out any non-error notifications first
+  await dismissNotifications(page)
+
+  const errorDetails = await page.$x('(//a | //*[@role="button"] | //button)[contains(normalize-space(.),"Details")]')
```

Comment:
TOL: This looks like it could use the `clickable` helper, but doing that would create a circular dependency which I'm guessing would cause problems. Not something we need to unravel here, but it seems like we have a couple different 'utility' files that could be consolidated, which could enable some code re-use.